packages/commonwealth/client/scripts/helpers/validateJson.ts

```diff
@@ -0,0 +1,159 @@
+import validator from 'is-my-json-valid';
+
+const schema = {
```

Comment:
is this a place where we will keep the schema? I mean, where is gonna be the single source of truth? If the schema gets changed, we need to adjust it here on the frontend? Also it would be nice to have some comment about version of that schema.

Reply:
there is an endpoint where the schema is stored and will be updated. For now I thought to put the schema here so you can test it and see if it works. But we should not call the api to validate in the client as this will result in a slow response.

The schema could be cached in the client at some point.

Reply:
makes sense!